README.md

```diff
@@ -1246,11 +1247,13 @@ Defaults:
   - `false` (default) - ignored defaults are not reported
   - `true` - if an ignored default is present, throw an error
   - `"log"` - if an ignored default is present, log warning
-- _strictKeywords_: report unknown keywords in schemas. Option values:
```

Comment:
This change is unrelated but appeared to be inconsistency in the docs

Reply:
sorry, not sure what was the inconsistency?

Reply:
"Option values:" is used in most options (if not all)...

Reply:
You're right, my mistake